str_analysis/data/known_pathogenic_variant_catalog__GRCh37__58_loci_with_offtargets.json

```diff
@@ -0,0 +1,2433 @@
+      "DiscoveryMethod": "L, CGA",
+      "DiscoveryYear": 2001,
+      "Diseases": [
+         {
+            "Symbol": "HPE5",
+            "Name": "Holoprosencephaly 5",
+            "Inheritance": "AD",
+            "OMIM": "609637",
+            "NormalMax": 15,
+            "PathogenicMin": 25
+         }
+      ],
+      "OfftargetRegions": [
+         "12:65957339-65957682",
+         "16:24740836-24741193",
+         "16:24788215-24788616",
+         "16:56849613-56849994",
+         "16:73580408-73580794",
+         "17:10898867-10899244",
+         "17:49908873-49909245",
+         "18:53253224-53253614",
+         "19:13318521-13318866",
+         "1:21502905-21503255",
+         "1:53793354-53793697",
+         "2:225449721-225450068",
+         "2:87141244-87141858",
+         "5:137929768-137930128",
+         "5:166406025-166406402",
+         "5:443175-443544",
+         "6:104467037-104467394",
+         "6:16327703-16328107",
+         "6:170870842-170871263",
+         "6:45390332-45390683",
+         "7:100291772-100292165",
+         "7:104654446-104654794",
+         "7:121928089-121928459",
+         "X:146993408-146993783",
+         "X:150151713-150152085",
+         "X:20008884-20009246",
+         "X:21385178-21385539",
+         "X:24382204-24382718",
+         "X:66764998-66765411"
+      ]
+   },
+   {
+      "LocusId": "ZIC3",
+      "LocusStructure": "(GCN)*",
+      "VariantType": "RareRepeat",
+      "ReferenceRegion": "X:136648985-136649015",
+      "RepeatUnit": "GCN",
+      "InExpansionHunterVariantCatalog": false,
+      "Gene": "ZIC3",
+      "Inheritance": "XR",
+      "GeneRegion": "exon - polyalanine",
+      "Diseases": [
+         {
+            "Symbol": "VACTERLX",
+            "Name": "X-linked VACTERL syndrome",
+            "Inheritance": "XR",
+            "OMIM": "314390",
+            "NormalMax": 10,
+            "PathogenicMin": 12
+         }
+      ],
+      "OfftargetRegions": [
+         "12:65957339-65957682",
+         "16:24740836-24741193",
+         "16:24788215-24788616",
+         "16:56849613-56849994",
+         "16:73580408-73580794",
+         "17:10898867-10899244",
+         "17:49908873-49909245",
+         "18:53253224-53253614",
+         "19:13318521-13318866",
+         "1:21502905-21503255",
+         "1:53793354-53793697",
+         "2:225449721-225450068",
+         "2:87141244-87141858",
+         "5:137929768-137930128",
+         "5:166406025-166406402",
+         "5:443175-443544",
+         "6:104467037-104467394",
+         "6:16327703-16328107",
+         "6:170870842-170871263",
+         "6:45390332-45390683",
+         "7:100291772-100292165",
+         "7:104654446-104654794",
+         "7:121928089-121928459",
+         "X:146993408-146993783",
+         "X:150151713-150152085",
+         "X:20008884-20009246",
+         "X:21385178-21385539",
+         "X:24382204-24382718",
+         "X:66764998-66765411"
+      ]
+   }
+]
```

Comment:
not sure if this matters for the json files, but GitHub thinks there should be a newline here